src/crimson/osd/osd_operations/background_recovery.h

```diff
@@ -38,16 +38,20 @@ class BackgroundRecoveryT : public PhasedOperationT<T> {
 private:
   virtual void dump_detail(Formatter *f) const;
   crimson::osd::scheduler::params_t get_scheduler_params() const {
+    int cost = std::max<int64_t>(1, (pg->get_info().stats.stats.sum.num_bytes / pg->get_info().stats.stats.sum.num_objects));
```

Comment:
Can we introduce get_average_object_size()?
W need to ensure num_objects and num_bytes are non-negative